packages/visx-demo/src/components/GalleryTile.tsx

```diff
@@ -28,21 +65,24 @@ export default function GalleryTile<ExampleProps extends WidthAndHeight>({
   tileStyles,
   title,
 }: Props<ExampleProps>) {
+  const { everVisible, ref } = useEverVisible();
   return (
     <>
       {renderLinkWrapper(
         exampleUrl,
-        <div className="gallery-tile" style={tileStyles}>
+        <div ref={ref} className="gallery-tile" style={tileStyles}>
           <div className="image">
-            <ParentSize>
-              {({ width, height }) =>
-                React.createElement(exampleRenderer, {
-                  width,
-                  height: height + (title || description ? detailsHeight : 0),
-                  ...exampleProps,
-                } as ExampleProps)
-              }
-            </ParentSize>
+            {everVisible && (
```

Comment:
If you went with my [suggestion][1], you'd want to change this to `entry?.isIntersecting`.

[1]: https://github.com/airbnb/visx/pull/1896/files#r1894403694